app/api/config.js

```diff
@@ -0,0 +1,9 @@
+﻿
+let config = {}
+
+config.api      = {};
+config.api.url  = process.env.API_URL || 'https://api.cbddev.xyz:443';
```

Comment:
May be we should trim the trailing / 
```suggestion
config.api.url  = (process.env.API_URL || 'https://api.cbddev.xyz:443').replace(/\/*$/, '')
```